cdcpd/src/cdcpd_node.cpp

```diff
@@ -183,6 +189,32 @@ CDCPD_Moveit_Node::CDCPD_Moveit_Node(std::string const& robot_namespace)
     }
   }
 
+void CDCPD_Moveit_Node::initialize_deformable_object_configuration(
+    Eigen::Vector3f const& rope_start_position, Eigen::Vector3f const& rope_end_position)
+{
+    if (node_params.deformable_object_type == DeformableObjectType::rope)
+    {
+        std::unique_ptr<RopeConfiguration> configuration = std::unique_ptr<RopeConfiguration>(
```

Comment:
prefer `std::make_unqiue` and `auto` to reduce having to write the type twice and have really long lines